nameparser/config/suffixes.py

```diff
@@ -6,6 +6,7 @@
     'esq',
     'esquire',
     'jr',
+    'jr.',
```

Comment:
I'd have to verify how you are processing the suffixes to respond with 100% certainty.  Since the suffixes are in a list, I think the code is most likely doing a set inclusion test, using the in operator.  Part of the answer depends on how you are parsing the text prior to doing the set inclusion test.  If your parsing picks up periods, then my answer is "yes" - you will need to have both the plain and period-ending versions of the suffixes.  Also, be aware that the set inclusion test is case sensitive.

In the new suffix preprocessing routine I added, the regex pattern has a "\.?", which will match the prior text with and without a trailing period character.  The question mark makes the period character optional.

Reply:
What do you want to happen?

The parsing code does a split on space characters, so any trailing punctuation (comma, period, semicolon, etc.) will remain with the word.

Reply:
Is period removal sufficient?  I'm thinking about other punctuation characters that may separate suffixes, such as comma.

Reply:
The first step of the parser is to split the entire string on commas. Many suffixes listed at the end of a name are always treated as suffixes. These are the 2 supported formats for commas:

Lastname [Suffix], Title Firstname (Nickname) Middle Middle[,] Suffix [, Suffix]
Title Firstname M Lastname [Suffix], Suffix [Suffix] [, Suffix]